dashboard/test/controllers/registrations_controller_test.rb

```diff
@@ -222,11 +232,13 @@ class RegistrationsControllerTest < ActionController::TestCase
     end
   end
 
+  # USING OLD SIGN-UP FLOW [START]
+
   test "teachers go to specified return to url after signing up" do
     session[:user_return_to] = user_return_to = '//test.code.org/the-return-to-url'
 
     assert_creates(User) do
-      post :create, params: {user: @default_params}
+      post :create, params: {user: @default_params.update(user_type: 'teacher', age: '21+', email_preference_opt_in: true)}
```

Comment:
I noticed this test was labeled as being for teachers but used the default params that use student as the user type so I updated that here.